courses/views.py

```diff
@@ -0,0 +1,25 @@
+"""Course API views"""
+from rest_framework import viewsets
+
+from courses import serializers, models
+
+
+class ProgramViewSet(viewsets.ModelViewSet):
+    """API view set for Programs"""
+
+    serializer_class = serializers.ProgramSerializer
+    queryset = models.Program.objects.all()
+
+
+class CourseViewSet(viewsets.ModelViewSet):
+    """API view set for Courses"""
+
+    serializer_class = serializers.CourseSerializer
+    queryset = models.Course.objects.all()
```

Comment:
I imagine we'll want to update/delete non-live courses etc. via the API, so I figured it would be best to leave that filter off for this initial implementation